src/csharp/Microsoft.Spark/Interop/Internal/Dotnet/ArrayExtensions.cs

```diff
@@ -0,0 +1,22 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Spark.Interop;
+using Microsoft.Spark.Interop.Internal.Java.Util;
+
+namespace System
+{
+    public static class ArrayExtensions
```

Comment:
Can we add a short description for this class and its functions?

Reply:
Emmm isn't the summary explaining it? I'm adding a class named `ArrayExtensions` to host extension methods for base class `System.Array` which is T[].